src/pages/CreateEvent/StorageUtils.tsx

```diff
@@ -0,0 +1,44 @@
+interface EventFormInputs {
```

Comment:
the fields description and thumbnail needs to also be updated to the new names here

Reply:
Done